### PR TITLE
feat(module:modal): add nzCentered for modal

### PR DESCRIPTION
**Pull request by lsbbd**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[x] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #6327 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/modal/doc/index.zh-CN.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ import { NzModalModule } from 'ng-zorro-antd/modal';
 | nzAfterClose      | Modal 完全关闭后的回调，可监听close/destroy方法传入的参数 | EventEmitter | - |
 | nzBodyStyle       | Modal body 样式 | `object` | - |
 | nzCancelText      | 取消按钮文字。<i>设为 null 表示不显示取消按钮（若在普通模式下使用了 nzFooter 参数，则该值无效）</i> | `string` | 取消 |
+| nzCentered        | 垂直居中展示 Modal | `boolean` | `false` |
```
Comment (hsuanxyz) [inline components/modal/doc/index.zh-CN.md:37]:
```suggestion
| nzCentered        | 居中展示 Modal | `boolean` | `false` |
```

**Review by hsuanxyz — CHANGES_REQUESTED** [review]

File: components/modal/doc/index.zh-CN.md (hunk 1 above)
Comment (lsbbd) [reply components/modal/doc/index.zh-CN.md:37]: @hsuanxyz The api description of `nzCentered` is actually a direct reference to the `react` version. Does it need to be unified here?

Comment (hsuanxyz) [reply components/modal/doc/index.zh-CN.md:37]: Ok, let's keep the way

**Review by hsuanxyz — APPROVED** [review]
LGTM

**Pull request merged**